Makefile

```diff
@@ -104,6 +110,30 @@ format: tidy ## Formats the code. Must have goimports and goimports-reviser inst
 dep: tidy ## Sorts dependencies
 	${OPTS} go mod vendor -v
 
+snapshot: sysroot ## create snapshot release
+	docker run --rm --privileged \
+		-v $(CURDIR):/go/src/github.com/skycoin/skywire \
+		-v /var/run/docker.sock:/var/run/docker.sock \
+		-v $(GOPATH)/src:/go/src \
+		-v $(CURDIR)/sysroot:/sysroot \
+		-w /go/src/github.com/skycoin/skywire \
+		alexadhyatma/golang-cross:$(GO_BUILDER_VERSION) --snapshot --skip-publish --rm-dist
+
+snapshot-clean: ## Cleans snapshot / release
+	rm -rf ./dist
+
+sysroot:
+	mkdir -p ./sysroot
+	@echo "getting sysroot for cross compilation"
+	if [[ ! -f /tmp/snapshot-05-12-2021.tar.gz ]]; then \
+  		curl -L -o /tmp/snapshot-05-12-2021.tar.gz "https://alexadhy-git.s3-ap-southeast-1.amazonaws.com/snapshot-05-12-2021.tar.gz"; \
```

Comment:
Can you elaborate on these links, what is the contents? Ideally I think everything should be self-contained or rely on public tools that are maintained by someone, and I believe this is self-hosted by you?

Reply:
It’s just a sysroot from a running ubuntu 20.04 container containing libgtk-3-dev and libappindicator3-dev and all of their dependencies: https://github.com/alexadhy/sysroot/blob/master/scripts/rsync.sh

I was hosting it with git-lfs previously but it was just too big, so for now i hosted it in my s3 bucket, would be nice to find other alternatives.